### DISCUSSION TITLE
Self hosted on DigitalOcean can't access budibase on mobile

### DISCUSSION THREAD

**Discussion by ihabmzaky** [Budibase/budibase#8036] [Help] [answered]
I have managed to self host budibase on digitalocean and changed the port number to 80.  I can access budibase through a windows web browser but get a timeout error when trying to reach the URL IP from mobile.  Any ideas why that is?

**Comment by melohagan** [comment, marked as answer]
Hey @ihabmzaky 

I don't have any ideas to be honest!

Is it the portal or builder you can't access, or published apps? 

What make and model is your mobile phone? Have you tried on any other devices?

**Reply by ihabmzaky** [reply]
For some strange reason, the country blocked the IP addresses from digitalocean **only** from mobiles.  I was able to login only using a vpn.

**Reply by melohagan** [reply]
That is quite unusual! Thanks for getting back 🙏
